stdlib/contrib/anaisdg/print/print.flux

```diff
@@ -0,0 +1,23 @@
+// Package print provides function for converting values into tables
+// ## Metadata
+// introduced: NEXT
+package print
+
+
+import "array"
+
+// print function converts other data types to a table
+// ## Parameters
+// - val: The value you want to "print".
+// - result_name: The name you want your table to have.
+// ## Examples
+//
+// ### Perform a linear regression on a dataset
+// ```no_run
+// import "contrib/anaisdg/print"
+// import "print"
+//
+// < value = (sampledata.float() |> findRecord(fn: (key) => true,idx: 0))._value
+// >  print.print(val: value, result_name: "extracted value")
+// ```
```

Comment:
- Wrong title.
- To generate the example output, this example needs to run.
- You shouldn't have the 2nd import statement.
- The input shorthand `<` needs to be a stream of tables. This one isn't. I'd use the sample data as input.

~~~suggestion
// ### Print a value extracted from a stream of tables
// ```
// import "contrib/anaisdg/print"
//
// value = (sampledata.float() |> findRecord(fn: (key) => true,idx: 0))._value
//
// >  print.print(val: value)
// ```
~~~

I think you could show a basic example of printing a literal and then show an example of printing an extracted scalar value. For printing just a basic value:

~~~js
// ```
// import "contrib/anaisdg/print"
//
// > print.print(val: 123.4)
// ```
~~~